tests/integration/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Is this tsconfig necessary? There is no typescript in `tests`.

Reply:
It's unnecessary. Added it when I debugging the test and Cypress kept shouting for a `tsconfig.json`. Removed.